packages/xo-web/src/xo-app/xosan/new-xosan.js

```diff
@@ -375,69 +417,19 @@ export default class NewXosan extends Component {
-                  <table className='table table-striped'>
-                    <thead>
-                      <tr>
-                        <th />
-                        <th>{_('xosanName')}</th>
-                        <th>{_('xosanHost')}</th>
-                        <th>{_('xosanSize')}</th>
-                        <th>{_('xosanUsedSpace')}</th>
-                      </tr>
-                    </thead>
-                    <tbody>
-                      {map(lvmsrs, sr => {
-                        const host = find(hosts, ['id', sr.$container])
-
-                        return (
-                          <tr key={sr.id}>
-                            <td>
-                              <input
-                                checked={selectedSrs[sr.id] || false}
-                                disabled={disableSrCheckbox(sr)}
-                                onChange={event => this._selectSr(event, sr)}
-                                type='checkbox'
-                              />
-                            </td>
-                            <td>
-                              <Link to={`/srs/${sr.id}/general`}>
-                                {sr.name_label}
-                              </Link>
-                            </td>
-                            <td>
-                              <Link to={`/hosts/${host.id}/general`}>
-                                {host.name_label}
-                              </Link>
-                            </td>
-                            <td>{formatSize(sr.size)}</td>
-                            <td>
-                              {sr.size > 0 && (
-                                <Tooltip
-                                  content={_('spaceLeftTooltip', {
-                                    used: String(
-                                      Math.round(
-                                        (sr.physical_usage / sr.size) * 100
-                                      )
-                                    ),
-                                    free: formatSize(
-                                      sr.size - sr.physical_usage
-                                    ),
-                                  })}
-                                >
-                                  <progress
-                                    className='progress'
-                                    max='100'
-                                    value={(sr.physical_usage / sr.size) * 100}
-                                  />
-                                </Tooltip>
-                              )}
-                            </td>
-                          </tr>
-                        )
-                      })}
-                    </tbody>
-                  </table>
+                  {srsOnSameHost && (
+                    <span className='text-warning'>
+                      <Icon icon='alarm' /> {_('xosanSrOnSameHostMessage')}
+                    </span>
```

Comment:
Put this below the table so it doesn't make it jump and it's in place of the suggestions, which seems clearer that it's an issue.